go.mod

```diff
@@ -44,6 +47,7 @@ require (
 	google.golang.org/grpc v1.22.0 // indirect
 	gopkg.in/square/go-jose.v2 v2.3.1
 	gopkg.in/yaml.v2 v2.2.3
+	gotest.tools v2.2.0+incompatible // indirect
```

Comment:
I did not found where the dependency is coming from, can you please check if `go mod tidy` would drop it?

Reply:
thanks for the heads up - yes `go mod tidy` cleaned it 👍